aviary/docs/user_guide/pre_mission_and_mission.ipynb

```diff
@@ -61,19 +82,56 @@
     "To do this, you can use the \"bus\" system.\n",
     "The bus system allows you to connect variables from the pre-mission system to the mission system based on what you specify.\n",
     "This is especially relevant when you're using external subsystems as core Aviary does not use the bus system internally.\n",
-    "The notion of the bus system is detailed more within the `SubsystemBuilderBase` docstrings.\n",
-    "\n",
+    "The notion of the bus system is detailed more within the `SubsystemBuilderBase` docstrings.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "remove-cell"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# Testing Cell\n",
+    "# ********NOTE******** #\n",
```

Comment:
@johnjasa @jdgratz10

Reply:
I believe based on the context it should be `bus_variables`, since this example is walking through how to use the bus. The `get_parameters` function is for variables that are dymos parameters during mission, while the `get_bus_variables` function is for variables that are indeed dymos parameters during the mission but that are also calculated during the pre-mission portion of the external subsystem. John, feel free to correct me if something has changed here, but that's the way it was back when I built the bus port initially.